crates/driver/example.solvers.toml

```
@@ -1,12 +1,12 @@
 [[solvers]]
 name = "mysolver" # Arbitrary name given to this solver, must be unique
-url = "http://localhost:1234/solve" # Endpoint of the solver
-absolute-slippage = 12 # Denominated in wei, optional
-relative-slippage = 0.1 # Percentage in the [0, 1] range
+endpoint = "http://0.0.0.0:33219/solve"
+absolute-slippage = "12" # Denominated in wei, optional
+relative-slippage = "0.1" # Percentage in the [0, 1] range
```

Not for this PR, but you could make this:
```suggestion
slippage = { relative = "12", absolute = "0.1" }
```

---

Neither do I. Let's keep it as is then.